src/ordered.rs

```diff
@@ -214,7 +214,7 @@ impl OrderedDocument {
     /// the correct type.
     pub fn get_f64(&self, key: &str) -> ValueAccessResult<f64> {
```

Comment:
One thing to note, I left these as `get_f64`, since they really are getting `f64` values, not `Bson::Double` values. I wasn't sure if this was exactly the right choice, and considered adding a `get_double` alias, or replacing this with `get_double`. In the end, I decided to just leave it as-is, but am open to suggestions.

cc @saghm

Reply:
I'm fine leaving it as-is as well